main.tf

```diff
@@ -382,7 +382,7 @@ resource "aws_cloudfront_distribution" "default" {
   viewer_certificate {
     acm_certificate_arn            = var.acm_certificate_arn
     ssl_support_method             = var.acm_certificate_arn == "" ? "" : "sni-only"
-    minimum_protocol_version       = var.minimum_protocol_version
+    minimum_protocol_version       = var.acm_certificate_arn == "" ? "TLSv1" : var.minimum_protocol_version # If cloudfront_default_certificate=true, only TLSv1 can be specified (see: https://docs.aws.amazon.com/cli/latest/reference/cloudfront/create-distribution.html).
```

Comment:
This should be documented in the `minimum_protocol_version` variable description, not fixed here. Fixing it here leaves users having set `minimum_protocol_version ="TLSv1.2"` thinking they have it set but secretly not.